internal/monitors/telegraf/monitors/winperfcounters/win_perf_counters_windows.go

```diff
@@ -17,16 +17,18 @@ import (
 // fetch the factory used to generate the perf counter plugin
 var factory = telegrafInputs.Inputs["win_perf_counters"]
 
-// Configure the monitor and kick off metric syncing
-func (m *Monitor) Configure(conf *Config) error {
+// GetPlugin takes a perf counter monitor config and returns a configured perf counter plugin.
+// This is used for other monitors based on perf counter that manage their own life cycle
+// (i.e. system utilization, windows iis)
+func GetPlugin(conf *Config) *telegrafPlugin.Win_PerfCounters {
 	plugin := factory().(*telegrafPlugin.Win_PerfCounters)
 
 	// copy top level struct fields
 	deepcopier.Copy(conf).To(plugin)
 
 	// Telegraf has a struct wrapper around time.Duration, but it's defined
 	// in an internal package which the gocomplier won't compile from
-	plugin.CountersRefreshInterval.Duration = time.Duration(conf.CountersRefreshInterval) * time.Second
+	plugin.CountersRefreshInterval.Duration = time.Duration(conf.CountersRefreshInterval) * time.Millisecond
```

Comment:
The description in the config struct says "number of nanoseconds".  Can you encode the unit in the config option name (e.g. `CountersRefreshIntervalMillis`) or use `time.Duration` which will be parsed as a string (see https://github.com/signalfx/signalfx-agent/blob/master/internal/monitors/traceforwarder/monitor.go#L41 for an example) so that the user can specify the units themselves.